apps/docs-v2/app/routes/_docs.recipes.typesafe-input.mdx

```diff
@@ -65,6 +65,7 @@ const MyInputImpl = forwardRef<HTMLInputElement, MyInputProps<string>>(
             "aria-invalid": !!field.error(),
             ...rest,
           })}
+          ref={ref}
```

Comment:
This should get passed into `getInputProps` rather than directly to the component.

Reply:
Check, even better. Hope this is ok.